### DISCUSSION TITLE
http://localhost:3000/models/detail?pageType=models&name=LiSiPS-SSE-PBEsol-model&id=22

### DISCUSSION THREAD

**Discussion by giscus** [yanyumei/giscus-test#13] [Announcements]
# http://localhost:3000/models/detail?pageType=models&name=LiSiPS-SSE-PBEsol-model&id=22

http://localhost:3000/models/detail?pageType=models&name=LiSiPS-SSE-PBEsol-model&id=22

**Comment by smallplum** [comment]
lalal

**Reply by smallplum** [reply]
哈哈哈哈

**Reply by yanyumei** [reply]
回复1111
